maven-shared-doc-resource/src/main/resources/markdown/download.md.vm

```diff
@@ -0,0 +1,54 @@
+---
+title: Download ${project.name} Source
+---
+#*
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+*#
+#set( $h = "#" )
+#set( $downloadDirectory = $project.properties.downloadDirectory )
```

Comment:
we need some documentation of the properties used by this template.

Reply:
we we make decision where we should put it we will add a documentation